test/shim/test_sequencer.py

```diff
@@ -16,14 +16,17 @@
 from shimmingtoolbox.simulate.numerical_model import NumericalModel
 from shimmingtoolbox.shim.sequencer import extend_slice
 from shimmingtoolbox.shim.sequencer import update_affine_for_ap_slices
+from shimmingtoolbox.utils import set_all_loggers
 
 import numpy as np
 import pytest
 import os
 import nibabel as nib
 import json
-from matplotlib.figure import Figure
+import logging
```

Comment:
library imports should come before individual functions are imported

Reply:
Resolved in 1a5ce6c.